.github/workflows/2-deploy-to-staging.yml

```
@@ -45,7 +45,7 @@ jobs:
         with:
           payload: |
             {
-              "text": "Staging deploy **${{ steps.pages-deployment.outcome }}** for: ${{ github.ref_name }}"
+              "text": "Deploy finished for ${{ github.ref_name }}: *${{ steps.pages-deployment.outcome }}*"
```

nit: I correlate `finished` with the release succeeding. Could we change to:
```suggestion
              "text": "Deploy for ${{ github.ref_name }} resulted in: *${{ steps.pages-deployment.outcome }}*"
```